docs/index.md

```diff
@@ -0,0 +1,7 @@
+# Welcome to Pangeo Docker Stacks documentation!
```

Comment:
'Pangeo Docker Stacks' or 'Pangeo Docker Images'?

Reply:
i'd say images to match the name of the repository... but i'm also not too wed to the name

Reply:
+1 to keeping it same as name of the repository, we can change the repo name later if needed.